src/thermal_throttle.c

```diff
@@ -0,0 +1,100 @@
+/**
+ * collectd - src/thermal_throttle.c
+ * Copyright (C) 2017      notbaab
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
```

Comment:
The project as a whole is moving more and more towards the [MIT license](https://opensource.org/licenses/MIT). Please consider if you'd be willing to license your work under that license, too.